helix-core/src/main/java/org/apache/helix/monitoring/mbeans/MessageQueueMonitorMBean.java

```diff
@@ -21,6 +21,7 @@
 
 import org.apache.helix.monitoring.SensorNameProvider;
 
+@Deprecated
```

Comment:
Please remove it. There should be no direct reference to the MBean class. All usage should be done through the MBeanServer, so it is automatically backward compatible, and this class shall be removed.

Reply:
Thanks for the comment, I deleted the interface.